src/frontend/src/components/warnBox.ts

```diff
@@ -0,0 +1,36 @@
+import { html, TemplateResult } from "lit-html";
+import { warningIcon } from "./icons";
+
+interface warnBoxProps {
+  title: string;
+  message: string | TemplateResult;
+  componentType: string;
+  slot?: TemplateResult;
+}
+
+export const warnBox = (
```

Comment:
Looking at the uses of this, and considering the question about about `div` vs. `aside` I wonder if it wouldn't be nicer/cleaner to have e.g. `warnBoxDiv` and `warnBoxAside` for instance, or even always have the `warnBox` be a `div` and let the caller wrap it in an `<aside></aside>` if necessary

Reply:
I like the idea of wrapping it into an aside. If it's alright with @meodai I can go ahead and make that change.